debug/fill-extrusion-querying.html

```diff
@@ -0,0 +1,1391 @@
+<!DOCTYPE html>
```

Comment:
we already have one of these called `extrusion-querying`

Why not add the new Geojson to that?

Reply:
Nice, I wasn't aware of that one, I'll just remove this new one.

Reply:
Updated in https://github.com/mapbox/mapbox-gl-js/pull/10347/commits/2b33470ff74c65b4b4a8960a61b2846b2ae2eaef.